vital/api/sleep.py

```diff
@@ -27,6 +27,25 @@ def get(
             },
         )
 
+    def get_sleep_with_stream(
+        self,
+        user_key: str,
```

Comment:
Why `user_key`? Seems we're using `user_id` elsewhere

Reply:
Good spot!